src/Cooked/MockChain.hs

```diff
@@ -3,14 +3,10 @@
 module Cooked.MockChain (module X) where
 
 import Cooked.MockChain.Balancing as X
-import Cooked.MockChain.BlockChain as X
-import Cooked.MockChain.Direct as X
+import Cooked.MockChain.BlockChain as X hiding (MockChainLogEntry)
+import Cooked.MockChain.Direct as X hiding (MockChainReturn)
```

Comment:
For now we don't have any standard as to how we build our re-export files, but we could homogenize them with explicit export lists. This should be done all over the codebase though and would require its own PR.

Reply:
Ok, I'll open an issue. Still, I think it is better to display what we export inside the module (where we can refer to the various definitions inside the file) rather than what we don't provide and is not defined in the file (re-export module) where we talk about it.